src/lib.rs

```diff
@@ -1,3 +1,106 @@
+//! Rustdis is a partial Redis server implementation intended purely for educational purposes.
+//!
+//! The primary goal of rustdis is to offer a straightforward and comprehensible implementation,
+//! with no optimization techniques to ensure the code remains accessible and easy to understand.
+//! As of now, rustdis focuses exclusively on implementing Redis' String data type and its
+//! associated methods. You can find more about Redis strings here: [Redis
+//! Strings](https://redis.io/docs/data-types/strings/).
```

Comment:
AFAIK markdown links should not be split into multiple lines.

```suggestion
//! associated methods. You can find more about Redis strings here:
//! [Redis Strings](https://redis.io/docs/data-types/strings/).
```